packages/playground/src/components/Input.vue

```diff
@@ -21,22 +22,33 @@ const props = defineProps<{
 }>();
 
 onMounted(() => {
-  const blockToolAdapter = new BlockToolAdapter(props.model, 0);
+  const caretAdapter = new CaretAdapter(document.body, props.model);
+  const blockToolAdapter = new BlockToolAdapter(props.model, caretAdapter, 0);
 
-  if (input.value !== null) {
-    blockToolAdapter.attachInput(createDataKey('text'), input.value);
+  if (input1.value !== null) {
+    blockToolAdapter.attachInput(createDataKey('text1'), input1.value);
 
     props.model.addEventListener(EventType.CaretManagerUpdated, (evt: CaretManagerCaretUpdatedEvent) => {
-      index.value = (evt.detail.index as TextIndex)[0];
+      index.value = evt.detail.index;
     });
   }
+
+  if (input2.value !== null) {
+    blockToolAdapter.attachInput(createDataKey('text2'), input2.value);
```

Comment:
can we get rid of call of "createDataKey" for tools developers? It would be more friendly to just pass string